### PR TITLE
Add classes

### PR DESCRIPTION
**Pull request by Rifat8080**
In this branch, I have:

Created class Person with the following:
Instance vars: @id, @name, and @age.
Constructor with name, age, and parent_permission as parameters. name and parent_permission are optional and have default values of "Unknown" and true.
Getters for @id, @name, and @age.
Setters for @name and @age.
Private method of_age? that returns true if @age is greater or equal to 18 and false otherwise.
Public method can_use_services? that returns true if a person is of age or has permission from parents.
Created class Student with the following:
Inherits from Person.
Constructor extends parent's constructor by adding @classroom and a parameter for it.
Method play_hooky that returns "¯\(ツ)/¯".
Created class Teacher with the following:
Inherits from Person.
Constructor extends parent's constructor by adding @specialization and a parameter for it.
Override can_use_services? so it always returns true.
Each class is saved in a separate file.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+I would like to thank microverse for giving me chance to work on this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (DammyShittu) [inline readme.md]: - [ ] Your link is not working. To be more professional, kindly fix it.

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```
Comment (DammyShittu) [inline student.rb]: - [ ] Kindly set `parent_permission` properly.

  ```
  super(age, name, parent_permission: parent_permission)
  ```

File: teacher.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```
Comment (DammyShittu) [inline teacher.rb]: - [ ] Parent_permission should be set properly for the super method.

  ```
  super(age, name, parent_permission: parent_permission)
  ```

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Rifat8080,

## Highlights
✔️ Private method in class

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — APPROVED** [review]
# Hi @Rifat8080 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Broken link is fixed ✔
- parent_permission value is properly assigned ✔
- Correct use of Git Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**